pyhive/hive.py

```diff
@@ -252,6 +258,41 @@ def _fetch_more(self):
         for row in response.results.rows:
             self._data.append([_unwrap_col_val(val) for val in row.colVals])
 
+    def poll(self, fetch_logs=False):
+        req = ttypes.TGetOperationStatusReq(
+            operationHandle=self._operationHandle
+        )
+        response = self._connection.client.GetOperationStatus(req)
+        _check_status(response)
+        
+        
+        
+        if fetch_logs:
+            req = ttypes.TFetchResultsReq(
+                operationHandle=self._operationHandle,
+                orientation=0,
+                maxRows=10000, 
+                fetchType=1
```

Comment:
could you add a comment that 1=logs? (unfortunately looks like there's no constant)